docs/userguide/about/install.md

```diff
@@ -507,6 +507,24 @@ pip install earth2studio[windgust-afno]
 uv add earth2studio --extra windgust-afno
 ```
 
+:::
+::::
+:::::
+:::::{tab-item} CBottleSR
```

Comment:
Re-name this to just CBottle since all cBottle interfaces will use this

Then add a "Notes: Additional dependencies needed for CBottle3D data source, CBottleInfill diagnostic and CBottleSR diagnostic.

Thanks!

Reply:
fixed